code/modules/projectiles/guns/manufacturer/frontier_import/ballistics.dm

```diff
@@ -0,0 +1,229 @@
+/obj/item/gun/ballistic/automatic/pistol/mauler
+	name = "Mauler machine pistol"
+	desc = "A full auto machine pistol originating from the Shoal. Insane stopping power, but mostly useless outside of CQC with unarmored targets. Chambered in 9mm."
+	icon = 'icons/obj/guns/manufacturer/frontier_import/48x32.dmi'
+	lefthand_file = 'icons/obj/guns/manufacturer/hunterspride/lefthand.dmi'
+	righthand_file = 'icons/obj/guns/manufacturer/hunterspride/righthand.dmi'
+	mob_overlay_icon = 'icons/obj/guns/manufacturer/frontier_import/onmob.dmi'
+
+	icon_state = "mauler"
+	item_state = "hp_generic"
+	w_class = WEIGHT_CLASS_NORMAL
+	mag_type = /obj/item/ammo_box/magazine/m9mm_mauler
+	fire_delay = 0.06 SECONDS
+
+	gun_firemodes = list(FIREMODE_FULLAUTO)
+	default_firemode = FIREMODE_FULLAUTO
+
+	spread = 25
+	spread_unwielded = 50
+	recoil = 1
+	recoil_unwielded = 4
+	fire_sound = 'sound/weapons/gun/pistol/mauler.ogg'
+
+	rack_sound = 'sound/weapons/gun/pistol/candor_cocked.ogg'
+
+	lock_back_sound = 'sound/weapons/gun/pistol/slide_lock.ogg'
+	bolt_drop_sound = 'sound/weapons/gun/pistol/candor_cocked.ogg'
+
+	load_sound = 'sound/weapons/gun/pistol/candor_reload.ogg'
+	load_empty_sound = 'sound/weapons/gun/pistol/candor_reload.ogg'
+	eject_sound = 'sound/weapons/gun/pistol/candor_unload.ogg'
+	eject_empty_sound = 'sound/weapons/gun/pistol/candor_unload.ogg'
+
+/obj/item/gun/ballistic/automatic/pistol/mauler/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/automatic_fire, 0.06 SECONDS)
+
+/obj/item/ammo_box/magazine/m9mm_mauler
+	name = "mauler machine pistol magazine (9mm)"
+	desc = "A long, 12-round magazine designed for the Mauler machine pistol. These rounds do okay damage, but struggle against armor."
```

Comment:
"A 12-round magazine designed for the Mauler machine pistol."

Don't really need to comment on the ammo performance- it's plain 9mm, and could be reloaded with any 9mm variant anyway.